core/drivers/scmi-msg/smt.c

```diff
@@ -172,8 +170,10 @@ void scmi_smt_threaded_entry(unsigned int channel_id)
 /* Init a SMT header for a shared memory buffer: state it a free/no-error */
 void scmi_smt_init_agent_channel(struct scmi_msg_channel *chan)
 {
-	COMPILE_TIME_ASSERT(SCMI_PLAYLOAD_MAX + sizeof(struct smt_header) <=
-			    SMT_BUF_SLOT_SIZE);
+	static_assert(SCMI_SEC_PAYLOAD_SIZE + sizeof(struct smt_header) <=
+		      SMT_BUF_SLOT_SIZE &&
+		      IS_ALIGNED(SCMI_SEC_PAYLOAD_SIZE, sizeof(uint32_t)));
+	assert(channel && smt_header);
```

Comment:
This can't compile in the commit "drivers: scmi-msg: rename secure buffer size macros"

Reply:
thanks.